loadtest/tests.go

```diff
@@ -151,6 +151,72 @@ func actionPost(c *EntityConfig) {
 	}
 }
 
+func actionJoinLeaveChannel(c *EntityConfig) {
+	// figure out my user id
+	var userId string
+	if user, resp := c.Client.GetUserByEmail(c.UserData.Email, ""); resp.Error != nil {
+		cmdlog.Errorf("Unable to get user by email. User: %v, Error: %v", c.UserData.Username, resp.Error.Error())
```

Comment:
Should there be a `return` here?

Reply:
Fixed in https://github.com/mattermost/mattermost-load-test/pull/52/commits/c473fabd9aaf6ebbf63b60ac27eb1e0f47d9dcc0